src/main/java/org/spongepowered/mod/mixin/world/MixinWorld.java

```diff
@@ -104,8 +104,7 @@ public BlockLoc getBlock(int x, int y, int z) {
         return new BlockWrapper(this, x, y, z);
     }
 
-    @Override
-    public Biome getBiome(Vector3d position) {
+    public BiomeType getBiome(Vector3d position) {
```

Comment:
Your pull request is adding a command block mixin. It is not updating the submodules and implementing dummy methods for it. It's no help to update it anyway since the methods you've added to fix compile errors will need to be implemented another time.
So basically updating the submodule has nothing to do with this PR and will likely cause conflicts with others.